src/main/java/rest/demo/controller/ToDoController.java

```diff
@@ -0,0 +1,60 @@
+package rest.demo.controller;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import rest.demo.model.ToDo;
+import rest.demo.repository.ToDoRepository;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * @author Marcin Kulik
+ */
+
+@RestController
+@Slf4j
+@RequestMapping("/api")
+public class ToDoController {
+
+    @Autowired
+    private ToDoRepository toDoRepository;
+
+    @GetMapping("/todos")
+    public ResponseEntity<List<ToDo>> getAll() {
+
+        if (!toDoRepository.findAll().isEmpty()) {
+            List<ToDo> toDoList = toDoRepository.findAll();
+            log.info("Showing all To Do Lists : {}", toDoList);

```

Comment:
you are using loggers as debuggers... not needed! 
Thou shalt only use loggers to log if something went wrong!